hook.sh

```diff
@@ -77,9 +77,13 @@ for ((i = 0; i < "${#acme_live[@]}"; i++));do
     acme_records+=',"'"${acme_live[$i]}"'"'
 done
 
+# Find minimum_ttl for the domain and use that instead of hardcoding a ttl.
+# This allows use on any deSEC domain (Letsencrypt ignores TTL > 60).
+minimum_ttl=$(curl "${args[@]}" -X GET "https://desec.io/api/v1/domains/$DEDYN_NAME/" | grep -o "\"minimum_ttl\":[0-9]*" | grep -o "[0-9]*")
```

Comment:
- Again, there are assumptions about JSON formatting here. To work around that, we can remove newlines from the `curl` output and then allow arbitrary whitespace before and after the colon (see my other content).
- If we replace the outside quotes with single quotes, the inner ones don't need escaping.

Reply:
Good catch! Thanks.